style.css

```diff
@@ -10,28 +10,31 @@
 
 .left {
 float: left;
-width: 50%;
+width: 40%;
 height: 100%;
 color: red;
+font-size: 14px;
 
 }
 .right {
 float: right;
-width: 50%;
+width: 60%;
 height: 100%;
+margin-top: 20px;
+
 }
 
 .backgr-gray {
-
-  background-color: lightslategrey;
+background-color: rgb(236, 237, 238);
 }
 
 .img {
- width: 60%;
- height: 60%;
-}
+ width: 70%;
+ height: 70%;
+ }
 
 p {
 text-align: justify;
-font-size:24px;
+font-size:14px;
+padding: 20px;
```

Comment:
it is bad practice to set padding for global p tag, this should be for class, but think if you can add padding to wrapper of the this p - '.right'?

Also global tags should be placed at beginning of CSS file.